db/migrate/20230106001924_add_job_id_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddJobIdToItems < ActiveRecord::Migration[7.0]
+  def change
+    add_column :items, :job_id, :integer
```

Comment:
If you have some spare time, you could give the column a more descriptive name, such as `cancellation_job_id`. If you're in a hurry because of the upcoming presentation, it's fine by me to leave this as it is.